torch/linalg/__init__.py

```diff
@@ -379,33 +409,36 @@
 """)
 
 eigh = _add_docstr(_linalg.linalg_eigh, r"""
-linalg.eigh(input, UPLO='L', *, out=None) -> (Tensor, Tensor)
+linalg.eigh(A, UPLO='L', *, out=None) -> (Tensor, Tensor)
 
-Computes the eigenvalue decomposition of a complex Hermitian or real symmetric matrix. Supports batched inputs.
+Computes the eigenvalue decomposition of a complex Hermitian or real symmetric matrix.
 
-For a complex Hermitian or real symmetric matrix :math:`A`, this is defined as
+Letting :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`,
+the eigenvalue decomposition of a complex Hermitian or real symmetric matrix
+:math:`A \in \mathbb{K}^{n \times n}` is defined as
 
 .. math::
 
-    A = V \operatorname{diag}(L) V^{\text{H}}
+    A = Q \operatorname{diag}(\Lambda) Q^{\text{H}}\mathrlap{\qquad Q \in \mathbb{K}^{n \times n}, \Lambda \in \mathbb{R}^n}
 
-where :math:`V^{\text{H}}` is the conjugate transpose when :math:`V` is complex, and the transpose when :math:`V` is real-valued.
-The matrix :math:`V` (and thus :math:`V^{\text{H}}`) is orthogonal in the real case,
-and unitary in the complex case.
+:math:`Q^{\text{H}}` is the conjugate transpose when :math:`Q` is complex, and the transpose when :math:`Q` is real-valued.
+:math:`Q` is orthogonal in the real case and unitary in the complex case.
 
-:attr:`input` is assumed to be Hermitian (resp. symmetric), but this is not checked internally.
-When :attr:`UPLO` is `'L'` (default), only the lower triangular part of each matrix is used in the computation.
-When :attr:`UPLO` is `'U'`, only the upper triangular part of each matrix is used.
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
 
-The returned decomposition is represented as a namedtuple `(eigenvalues, eigenvectors)`,
-which corresponds to :math:`L`, :math:`V` above. If :attr:`input` is a batch of matrices,
-then `L`, `V` are also batched with the same batch dimensions.
+:attr:`A` is assumed to be Hermitian (resp. symmetric), but this is not checked internally, instead:
 
-The eigenvalues are returned in ascending order.
+- If :attr:`UPLO` `= 'L'` (default), only the lower triangular part of the matrix is used in the computation.
+- If :attr:`UPLO` `= 'U'`, only the upper triangular part of the matrix is used.
+
+The returned decomposition is represented as a namedtuple `(eigenvalues, eigenvectors)`,
```

Comment:
Think about moving this to a "returns" section

Reply:
Done!